src/main/java/com/google/sps/data/OrderHandler.java

```diff
@@ -152,16 +154,40 @@ Entity getClosestLibrary(List<Entity> libraries, Point address) throws ApiExcept
   }
 
   /**
-   * Get unassigned orders from datastore with the area set to area.
-   * TODO[ak47na]: change method to return a list of Order objects so that classes that call it 
-   * don't need to know how data is stored.
+   * Returns the keyString of unassigned orders from datastore with the area set to area.
    */
-  List<Entity> getAvailableOrders(int area) {
+  List<String> getAvailableOrders(int area) {
```

Comment:
You changed this method (good), but no tests needed changing (bad?).

I'm guessing this method is not tested?